w3ctrace/state_test.go

```diff
@@ -30,6 +31,18 @@ func TestParseState(t *testing.T) {
 			Header:   "rojo=00f067aa0ba902b7,    ,,congo=t61rcWkgMzE",
 			Expected: w3ctrace.State{"rojo=00f067aa0ba902b7", "congo=t61rcWkgMzE"},
 		},
+		"with 33 list items": {
+			Header:   strings.TrimLeft(strings.Repeat("rojo=00f067aa0ba902b7,", 33), ","),
```

Comment:
```suggestion
			Header:   strings.TrimRight(strings.Repeat("rojo=00f067aa0ba902b7,", 33), ","),
```
Otherwise it's leaving the trailing comma at the end, which is not an issue for the test, but contradicts the test name

Reply:
Good catch - typo 👍